integration-tests/package.json

```diff
@@ -50,7 +52,7 @@
       "/test/"
     ],
     "testPathIgnorePatterns": [
-      "./ledger-signer.spec.ts",
```

Comment:
should not be remove, we don't want those tests to run in the pipeline as it required interaction with a ledger